### PR TITLE
Remove burned NFTs from creation list

### PR DESCRIPTION
**Pull request by mikesizz**
This PR will check for the owners of all user's creations and remove any that belong to the burn address.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ const cors = require('cors')
 const _ = require('lodash')
 const conseilUtil = require('./conseilUtil')
 const { random } = require('lodash')
+
+const BURN_ADDRESS = 'tz1burnburnburnburnburnburnburjAYjjX'
```
Comment (mikesizz) [inline index.js:9]: Can move this to env file if desired.

**Comment by mikesizz** [comment]
If you can't un burn an NFT it may be a good idea to just update a file in s3 with all the objkt IDs that are burned. This would save on network request overhead. Would probably just store them in memory but this seems to be made to run in a server less environment.

**Comment by skenaja** [comment]
Won't this remove the obkjt from the creator's creation list if not all of the editions are owned by `burn`?

https://github.com/hicetnunc2000/hicetnunc/issues/280

**Comment by mikesizz** [comment]
@skenaja Yeah, i had asked about this on discord. I'll adjust the code.

**Comment by mikesizz** [comment]
@skenaja The newest commit will ensure that new NFTs that are burned will not show up in the creation list (by making sure all of them belong to the burn address). What is the desired outcome if an individual owns an objkt and the issuer burns their remaining issues?

**Pull request merged**